From 9a7652a311d6e41721831b93bd8b554a768421d7 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 12:04:27 +0800
Subject: [PATCH] Test case5-E for Add PCD_INFO_GENERATION under Define section

Description: Add PCD_INFO_GENERATION under Define section ("PCD_INFO_GENERATION = TRUE") in TestPkg/TestPkg32X64.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/TestPkg32X64.dsc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/TestPkg/TestPkg32X64.dsc b/TestPkg/TestPkg32X64.dsc
index 532d084223..5b614ad598 100644
--- a/TestPkg/TestPkg32X64.dsc
+++ b/TestPkg/TestPkg32X64.dsc
@@ -23,6 +23,7 @@
   BUILD_TARGETS                  = NOOPT|DEBUG|RELEASE
   SKUID_IDENTIFIER               = DEFAULT
   FLASH_DEFINITION               = TestPkg/TestIa32X64.fdf
+  PCD_INFO_GENERATION            = TRU
 
   #
   # Defines for default states.  These can be changed on the command line.
-- 
2.27.0.windows.1

